src/task16/task16

```diff
@@ -0,0 +1,29 @@
+class MultiplicatorUnitFailure extends Error { }
+class InvalidInput extends Error { }
+
+var primitiveMultiply = (a, b) => {
+    if (Math.random() < 0.2) {
+        return a * b;
+    } else {
+        throw new MultiplicatorUnitFailure("Klunk");
```

Comment:
what is "Klunk"?

Reply:
yeah, it's also from the task
because I've done it from the book online
![image](https://user-images.githubusercontent.com/75950794/223053483-d4dc5c7b-846f-46dc-816c-890cec64b764.png)
they are called the same, but the content's a bit different 
will check it next time